net.zdechov.app.C-evo.yml

```diff
@@ -0,0 +1,50 @@
+app-id: net.zdechov.app.C-evo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: c-evo
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - name: c-evo
+    sources:
+      - type: svn
+        url: https://svn.zdechov.net/c-evo/tags/1.3.6
+        revision: r634
```

Comment:
this is an invalid revision

Reply:
>           revision (string)
>               A specific revision number to use

This is not a number you put.

Reply:
Revision number in terms of Subversion is both r123 and 123. From their manual https://svnbook.red-bean.com/en/1.7/svn.tour.revs.specifiers.html "Referring to revision numbers with an “r” prefix (r314, for example) is an established practice in Subversion communities, and is both supported and encouraged by many Subversion-related tools. In most places where you would specify a bare revision number on the command line, you may also use the rNNN syntax."

Apparently flatpak builder accepts both r123 and 123. So I guess that it is just your review observation and it is not based on any real check. Also flatpak-builder-lint doesn't complain that it is not just a number.

Anyway I have updated that as both works ok.